docs/quickstart.md

```diff
@@ -2,7 +2,7 @@ If you want to create Autonomous Economic Agents (AEAs) that can act independent
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/mwkAUh-_uxA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
This video is quite outdated at this point. I suggest we remove it from the documentation.

Reply:
We will update it soon